### PR TITLE
Fix "less than" format amount bug

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
* **#341** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/341?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/next/pages/activity.tsx [hunk 1]
```diff
@@ -19,9 +19,7 @@ export const Page: NextPageWithLayout = () => {
 export const getServerSideProps = userProtectedGetSSP()
 
 Page.getLayout = (children) => (
-  <HomeLayout TopNav={<TopNav header="Activity" button={ButtonOption.NONE} />}>
-    {children}
-  </HomeLayout>
+  <HomeLayout TopNav={<TopNav header="Activity" />}>{children}</HomeLayout>
```
Comment (0xBigBoss) [inline apps/next/pages/activity.tsx:22]: why remove this?

**Review by 0xBigBoss — APPROVED** [review]
lgtm, but double check the removed code on the activity page first.

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Apr 22, 8:26 PM EDT**: @youngkidwarrior merged this pull request with [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/341).

**Pull request merged**